Cloud Application Manager/Automating Deployments/terraform-template-box.md

```diff
@@ -7,17 +7,17 @@
 }}}
 **In this article:**
 
-* [Overview](#overview)
-* [Audience](#audience)
-* [Prerequisites](#prerequisites)
-* [Create a Terraform Template and apply configuration](#create-a-terraform-template-and-apply-configuration)
-* [Update a Terraform configuration in Real-Time](#update-a-terraform-configuration-in-real-time)
-* [Connect to Other Cloud Application Manager Boxes over Bindings](#connect-to-other-cloud-application-manager-boxes-over-bindings)
-* [Getting General Support](#getting-general-support)
+- [Overview](#overview)
+- [Audience](#audience)
+- [Prerequisites](#prerequisites)
+- [Create a Terraform Template and apply configuration](#create-a-terraform-template-and-apply-configuration)
+- [Update a Terraform configuration in Real-Time](#update-a-terraform-configuration-in-real-time)
+- [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
```

Comment:
Why you changed this one? Now there are two supports, and no bindings section

Reply:
Hola Guillermo,

I use Visual Studio Code to edit Knowledge Base articles. The only reason I use it is that it has a TOC generation plugin. I regenerate the TOC every time I modify a document, to make sure it is accurate. (Now I've managed to set up TOC formatting correctly.)

After digging in the history, it seems that someone forgot to update TOC some time ago. There is no section about bindings in upstream/master.

-  Do you want me to remove the "Contaction Cloud Application Manager Support" section from the article?

- Which file serves as a standard example to copy headers, overview, audience, prerequisites and support sections from?

Thank you.

Reply:
Hola @DenesPal,

Yes, you need to configure the plugin options to specify the ToC bullet points symbol to be used. There is another plugin that I use that is a sort of lint for MarkDown, which notifies you of some of these formatting issues.

Regarding the section, @EfrenRey may confirm if the bindings section was required (although it had no content, so it should be added), or if it was just added but not required). Additionally Denes, I saw that with your change there were two support sections, which I think that you have already fixed. Thanks. @EfrenRey, could you confirm if we need to document the bindings support for Terraform? Thanks!

Reply:
I've removed old support section now.